api/v1beta1/probe.go

```diff
@@ -23,8 +23,16 @@ type KogitoProbe struct {
 	// +optional
 	LivenessProbe corev1.Probe `json:"livenessProbe,omitempty"`
 
+	// HTTP path to check for liveness. Defaults to "q/health/live".
+	// +optional
+	LivenessPath string `json:"livenessPath,omitempty"`
```

Comment:
Hey @Kevin-Mok! I agree with @sutaakar. We don't need this parameter since the `LivenessProve` object already have a `path` attribute that can be used:
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request

We just have to keep in our documentation a section where we refer to this approach and how a user could change the defaults. I've updated the JIRA with this information. 

Many thanks!